__tests__/app.test.js

```diff
@@ -92,6 +92,65 @@ describe("GET review by id", () => {
   });
 });
 
+describe("PATCH review votes", () => {
+  test("200: should accept an object in the form { inc_votes: newVote } and successfully update the database returning the updated review to the user when increasing vote count", () => {
+    const votesToPatch = { inc_votes: 6 };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(votesToPatch)
+      .expect(200)
+      .then((res) => {
+        expect(res.body.updatedReview.votes).toBe(11);
+      });
+  });
+  test("200: should accept an object in the form { inc_votes: newVote } and successfully update the database returning the updated review to the user when decreasing vote count", () => {
+    const votesToPatch = { inc_votes: -4 };
+    return request(app)
+      .patch("/api/reviews/2")
```

Comment:
It would be good to amend this test to check a - vote number on a different article_id to the previous test, this will give increased confidence that your vote change functionality works on different articles.